zstacklib/zstacklib/utils/objgraph.py

```
@@ -0,0 +1,960 @@
+
+    .. versionchanged:: 2.1
+       New parameter: ``file``.
+
+    """
+    stats = most_common_types(limit, objects, shortnames=shortnames)
+    width = max(len(name) for name, count in stats)
+    for name, count in stats:
+        file.write('%-*s %i\n' % (width, name, count))
+
+
+def show_growth(limit=10, peak_stats={}, shortnames=True, file=sys.stdout):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)